submissions/yhorodechna/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/phone.png" alt="phone picture" />
+                            <span>Phone</span>
+                        </li>
+                        <li class="popup__more more">
+                            <input tabindex="2" class="more__input" type="checkbox" id="more__input" />
+                            <label class="more__label" for="more__input"><span class="more__btn"></span></label>
+                            <ul class="more__list">
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/picture.png" alt="picture" />
+                                    <span>Picture</span>
+                                </li>
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/plane.png" alt="plane picture" />
+                                    <span>Plane</span>
+                                </li>
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/shopping-cart.png" alt="shopping picture" />
+                                    <span>Shopping</span>
+                                </li>
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/write.png" alt="write picture" />
+                                    <span>Note</span>
+                                </li>
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/responsive.png" alt="responsive picture" />
+                                    <span>Responsive</span>
+                                </li>
+                                <li class="popup__item" tabindex="3">
+                                    <img src="img/statistics.png" alt="statistics picture" />
+                                    <span>Statistics</span>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="menu">
+                <ul>
+                    <li class="menu__item" tabindex="4"><img src="img/header/search.png" alt="search picture" />
+                    </li>
+                    <li class="menu__item" tabindex="4"><img src="img/header/notification.png"
+                            alt="notification picture" /></li>
+                    <li class="menu__item" tabindex="4"><img src="img/header/user.png" alt="user picture" /></li>
+                    <li class="menu__item" class="test" tabindex="4"><img src="img/header/settings.png"
+                            alt="settings picture" />
+                    </li>
+                </ul>
+            </div>
```

Comment:
If this block a menu why you don't use `nav` tag?

Reply:
I missed it . Solved.